mvc/migration/migratingfrommvc5.rst

```diff
@@ -1,257 +1,244 @@
 Migrating From ASP.NET MVC 5 to MVC 6
 =====================================
 
-By `Steve Smith`_ 
+By `Rick Anderson`_ and `Steve Smith`_
 
-Migrating from ASP.NET MVC 5 to ASP.NET 5 and MVC 6 requires a few steps to complete, since ASP.NET 5 introduces a number of new concepts. In this article you will learn how to migrate from the ASP.NET MVC 5 default project template to ASP.NET MVC 6, including initial setup, basic controllers and views, static content, and client side dependencies.
+While this article shows how to migrate an ASP.NET MVC 5 project to ASP.NET MVC 6, it's arguably more valuable in showing what has changed from MVC 5 to MVC 6. The migration process in this (part 1 tutorial) includes initial setup, basic controllers and views, static content, and client side dependencies. The tutorial creates the MVC 6 app with the empty template and adds all the assets from the legacy MVC 5 app needed to complete the conversion. An alternative approach to migrations is to create the MVC 6 app with the full MVC app template, and then copy the assets and your custom code.
 
-In this article:
-	- `Create the Initial Project`_
-	- `Create the Destination Solution`_
-	- `Migrate Basic Controllers, Views, and Static Content`_
-	- `Configure Bundling`_
+	- `Create the starter MVC 5 project`_
+	- `Create the MVC 6  project`_
+	- `Configure the site to use MVC`_
+	- `Controllers and views`_
+	- `Static content`_
+	- `Migrate the layout file`_
 
-Download the `finished source <https://github.com/aspnet/Docs/tree/master/samples/MigratingFromMvc5/NewMvc6Project>`_ from the project created in this article.
+Create the starter MVC 5 project
+========================================
 
-Create the Initial Project
---------------------------
-
-For the purposes of this article, we will be starting from the default ASP.NET MVC 5 starter web project, which you can create in Visual Studio 2015 by adding a new web project and choosing MVC 5.
+To demonstrate the upgrade, we'll start by creating an new ASP.NET MVC 5 app. Create it with the name *WebApp1* so the namespace will match the MVC 6 project we create in the next step.
```

Comment:
Should not be using "we'll", you should use "you'll". Same goes with "we" -> "you".